src/v1/util/RegistrationFormFactory.js

```diff
@@ -86,8 +86,10 @@ const checkSubSchema = function(subSchema, value, model) {
         // with email as login enabled, we only have email populated
         // Therefore we fallback and run validation with email attribute.
         fieldValue = model.has('userName') ? model.get('userName') : model.get('email');
+        return !passwordContainsFormField(fieldValue, password);
+      } else {
+        return !fieldValue || password.indexOf(fieldValue) === -1;
```

Comment:
was the logic before only checking if the username is part of the password and not checking first and last name at all?

Reply:
see attached snippet for detail https://github.com/okta/okta-signin-widget/blob/5183778d9b81c02e1e28cea8a04e463ae7961d38/src/v1/util/RegistrationFormFactory.js#L19-L59